build-scripts/prepare_environment.sh

```diff
@@ -10,22 +10,15 @@
 set -ex
 printenv
 
-if [ $1 = "cdh5" ]; then
```

Comment:
what happened to this? we still need this for the client bundles. this script gets called in all of the builds, not just the assembly.

Reply:
Oh okay, I thought I could clean it up. Restored.

Reply:
i appreciate the sentiment, it is definitely ugly.  unfortunately i think we can only clean it up once we do the hadoop3 client upgrade. thanks